packages/terra-arrange/src/Arrange.module.scss

```diff
@@ -7,8 +7,6 @@
   // Vertically aligns arrange content to the top by default
   .arrange {
     display: flex;
-    max-width: 100%;
```

Comment:
is these styles used in source code ? if yes, have we seen the impact of this removal in all the components of terra framework where this component is referred ?

Reply:
Yes, these styles are used in source code. But these styles are added recently to fix the zoom reflow issue in mobile view. I have verified in terra core and terra framework, its working as expected even though we removed these styles.